lib/src/simple_op_store.rs

```diff
@@ -122,7 +122,7 @@ fn upgrade_to_thrift(store_path: PathBuf) -> std::io::Result<()> {
 
     fs::write(tmp_store_path.join("thrift_store"), "")?;
     let backup_store_path = store_path.parent().unwrap().join("op_store_old");
-    fs::rename(&store_path, &backup_store_path)?;
+    fs::rename(&store_path, backup_store_path)?;
```

Comment:
Interestingly, Clippy didn't find this instance for me, but it found one on src/file_util.rs:115 (the `&target` argument there).

Reply:
Weird. My clippy doesn't find that. I'll fix that too.